### DISCUSSION TITLE
New user in need of guidance!

### DISCUSSION THREAD

**Discussion by josephsdavid** [nvim-neorg/neorg#203] [Q&A] [answered]
Hi all! I am super excited about norg, however I have some pretty basic questions. 1, how do i manage a bunch of notes in a workspace? for example, if I had `main_dir`, with an overall norg file, and within that a bunch of projects with their own notes, would i open the main notes in that workspace and add to that, or could i link them all together? Secondarily, and more basic, how do i display a table of contents, and how do I jump to a linked item (definition, general link, another file?)

**Comment by mrossinek** [comment, marked as answer]
Hi @josephsdavid! Awesome to hear that you're excited about neorg!

1. I am not 100% sure what you're getting at tbh.. Do you mean in terms of navigating between your notes in that nested directory structure? For that I would suggest using some sort of file picker or file explorer (heck, even netrw could suffice). Whatever you feel comfortable with and what fits your usual workflow.
If you're more asking about linking notes to one another, there is a way to link to another norg file with no specific target. The syntax for this is not on the `main` branch yet but will become available once we merge #178. The syntax is the following: `{:some_other_file:}`. There are also some options for specifying paths relative to other workspaces, global paths, etc.
We are planning to add some [Zettelkasten](https://zettelkasten.de/posts/overview/) module in the future but nothing is in the works right now.

2. There are multiple questions here.
- previewing the Table of Contents will become available once we merge #159. We have not yet defined a default key binding for that.
- the default key binding for following link is to simply hit "Enter" (vim notation: `<CR>`) while you're cursor is above the link you wish to follow. There are some odd edge cases/bugs with this on `main` which, as before, will be solved by #178.

**Reply by josephsdavid** [reply]
Hah! Sorry first question is a little complicated and poorly worded, but you answered it! And my long standing vim question about opening links! Appreciate it!

**Reply by mrossinek** [reply]
Just to clarify: using `<CR>` to follow links is not a vim default. I only mentioned that notation in case you want to modify that key binding and wonder how to specify the "Enter" key in the config :+1:

**Reply by josephsdavid** [reply]
That was exactly my plan! Thanks again
